src/components/GenreSelect/GenreSelect.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import GenreSelect from './GenreSelect';
+
+describe('GenreSelect', () => {
+    it("should render all genres passed in props'", () => {
+        const listGenres = ['Action', 'Adventure', 'Comedy'];
+        render(<GenreSelect listGenres={listGenres} selectedGenre='' onSelect={() => {}} />);
+
```

Comment:
You can use destructuring here like const {getByText} =  render(<GenreSelect listGenres={listGenres} selectedGenre='' onSelect={() => {}} />);